README.md

```diff
@@ -2,41 +2,50 @@
 <a href="https://github.com/salesforce-ux/design-system-ui-kit/archive/master.zip"><img src="https://user-images.githubusercontent.com/1750832/41082861-8013ecba-69e4-11e8-8149-7eaa94825b19.png" alt="Design System UI Kit" /></a>
 </p>
 <br />
-<h1 align="center" style="border-bottom:none;">Salesforce Lightning Design System (SLDS) UI Kit</h1>
-<p align="center">
-The SLDS UI Kit is a collection of Sketch Libraries of <a href="https://www.lightningdesignsystem.com">Lightning Design System</a> components and design patterns. The thousands of customers and partners who build applications on the Salesforce platform can use these resources to quickly visualize and test designs before they're implemented in their Salesforce organizations.
+<h1 style="border-bottom:none;">Salesforce Lightning Design System (SLDS) UI Kit</h1>
+<p>
+The Salesforce Lightning Design System (SLDS) UI Kit is a collection of resources to support designing and prototyping using <a href="https://www.lightningdesignsystem.com">Lightning Design System</a> components, tokens, and design patterns. There are also useful resources to help make design workflows more efficient with artifacts like page templates, wireframes, key product screens, and components for writing specifications. 
 </p>
-<p align="center">Install and always stay up to date by subscribing to the Sketch Library links below.
-<br />
-<strong>» Copy/paste the URLs below in any browser address bar «</strong></p>
-<ol>
-<li><b>Component Library</b>
+<strong>» To install, copy/paste the URLs below in any browser address bar «</strong></p>
+<i>Required libraries</i>
+<ul>
+<li><b>SLDS Icons</b>
```

Comment:
We should be consistent with the use of strong vs b if possible. Also, if we are bolding or italicizing to act as a header, h tags might be better from an accessibility standpoint

Reply:
Updated!